### PR TITLE
11.40 - Monsters

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Add all monsters from update 11.40

[cont]: https://github.com/EPuncker/orts2/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: data/monster/monsters/animated_feather.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		<flag canwalkonpoison="1" />
+	</flags>
+	<immunities>
+		<immunity paralyze="1" />
+		<immunity invisible="1" />
+		<immunity drunk="1" />
+	</immunities>
+	<elements>
+		<element firePercent="-18" />
+		<element icePercent="100" />
```
Comment (torresmon) [inline data/monster/monsters/animated_feather.xml:23]: <immunity ice="1" />

Comment (torresmon) [inline data/monster/monsters/animated_feather.xml:23]:
```suggestion
		<immunity ice="1" />
```

Comment (torresmon) [inline data/monster/monsters/animated_feather.xml:23]: trocar por `<immunity ice="1" />` ?

File: data/monster/monsters/animated_ogre_brute.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		<flag canpushcreatures="0" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="0" max="-269" fire="6" />
+		<attack name="fire" interval="2000" chance="15" range="7" min="-70" max="-180">
```
Comment (torresmon) [inline data/monster/monsters/animated_ogre_brute.xml:18]: ta certo esse name="fire" ? o shooteffect tá como poison

Comment (torresmon) [inline data/monster/monsters/animated_ogre_brute.xml:18]: o brute taca um boulder de dano físico

Comment (torresmon) [inline data/monster/monsters/animated_ogre_brute.xml:18]:
```suggestion
		<attack name="boulder" interval="2000" chance="15" range="7" min="0" max="-100">
			<attribute key="shootEffect" value="largerock" />
			<attribute key="areaEffect" value="explosionarea" />
```

File: data/monster/monsters/animated_ogre_brute.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			<attribute key="shootEffect" value="poison" />
+		</attack>
+		<attack name="drunk" interval="2000" chance="10" range="7">
+			<attribute key="shooteffect" value="energy" />
+			<attribute key="areaeffect" value="teleport" />
+		</attack>
+	</attacks>
```
Comment (torresmon) [inline data/monster/monsters/animated_ogre_brute.xml:25]:
```suggestion
		<attack name="speed" interval="2000" chance="15" radius="6" target="0" speedchange="-800" duration="15000">
			<attribute key="areaEffect" value="redshimmer" />
		</attack>
	</attacks>
```

File: data/monster/monsters/animated_ogre_brute.xml (hunk 2 above)
Comment (torresmon) [inline data/monster/monsters/animated_ogre_brute.xml:18]:
```suggestion
		<attack name="physical" interval="2000" chance="20" range="7" min="0" max="-100">
			<attribute key="shootEffect" value="largerock" />
		</attack>
```

File: data/spells/spells.xml [hunk 4]
```diff
@@ -836,4 +836,10 @@
 	<instant name="ferumbras soulfire" words="###66" aggressive="1" blockwalls="1" needtarget="1" needlearn="1" script="monster/ferumbras_soulfire.lua" />
 	<instant name="glooth fairy skill reducer" words="###67" direction="1" aggressive="1" blockwalls="1" needtarget="0" needlearn="1" script="monster/glooth_fairy_skill_reducer.lua" />
 	<instant name="physical explosion" words="###68" aggressive="0" blockwalls="1" needtarget="0" needlearn="1" script="monster/physical_explosion.lua" />
+	<instant name="freed soul spell" words="###69" direction="1" aggressive="1" blockwalls="1" needtarget="0" needlearn="1" script="monster/freed_soul_spell.lua" />
+	<instant name="heal monster 9x9" words="###70" direction="1" aggressive="0" blockwalls="1" needtarget="0" needlearn="1" script="monster/heal_monsters_9x9.lua" />
+	<instant name="heal monster" words="###71" direction="1" aggressive="0" blockwalls="1" needtarget="0" needlearn="1" script="monster/heal_monsters.lua" />
+	<instant name="pixie skill reducer" words="###72" blockwalls="1" aggressive="1" needtarget="0" exhaustion="2000" needlearn="1" script="monster/pixie_skill_reducer.lua" />
```
Comment (EPuncker) [inline data/spells/spells.xml:842]: tira o exaustion

File: data/spells/scripts/monster/freed_soul_spell.lua [hunk 5]
```diff
@@ -0,0 +1,42 @@
+local combat = Combat()
+combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_ENERGYDAMAGE)
+combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_PURPLEENERGY)
+
+combat:setArea(
+    createCombatArea(
```
Comment (EPuncker) [inline data/spells/scripts/monster/freed_soul_spell.lua:6]: trocar todos espaços por tab

**Pull request merged**